stubs/aiofiles/aiofiles/threadpool/binary.pyi

```diff
@@ -34,8 +36,15 @@ class AsyncBufferedIOBase(_UnknownAsyncBinaryIO):
     @property
     def raw(self) -> FileIO: ...
 
+class AsyncIndirectBufferedIOBase(AsyncIndirectBase[bytes], AsyncBufferedIOBase): ...
+
 class AsyncBufferedReader(AsyncBufferedIOBase):
     async def peek(self, __size: int = ...) -> bytes: ...
 
+class AsyncIndirectBufferedReader(AsyncIndirectBase[bytes], AsyncBufferedIOBase):
+    async def peek(self, __size: int = ...) -> bytes: ...
+
 class AsyncFileIO(_UnknownAsyncBinaryIO):
     async def readall(self) -> bytes: ...
+
+class AsyncIndirectFileIO(AsyncIndirectBase[bytes], AsyncFileIO): ...
```

Comment:
Similar comment to the one I made about `AsyncIndirectBufferedIOBase` -- I'm wary of inheriting from `AsyncFileIO` here.